pages/common/vimdiff.md

```diff
@@ -1,16 +1,36 @@
 # vimdiff
 
-> Open up to four files in vim and show the differences between them.
-> See `vim` documentation for working with files and navigating within a window.
+> Open up two or more files in vim and show the differences between them.
+> More information: https://www.vim.org.
 
-- Open two files and show the differences (up to four files can be compared):
+- Open two files and show the differences:
 
 `vimdiff {{file1}} {{file2}}`
 
-- Open two files using a horizontal window split instead of the default vertical split:
+- Move the cursor to the window on the left|right:
 
-`vimdiff -o {{file1}} {{file2}}`
+`Ctrl + w {{h|l}}`
 
-- Move the cursor to the window on the left|right|up|down:
+- Jump to next difference:
 
-`Ctrl + w {{h|l|k|j}}`
+`[c`
+
+- Jump to previous difference:
+
+`]c`
+
+- Obtain difference from other window to current window:
+
+`do`
+
+- Put difference from current window to other window:
+
+`dp`
+
+- Update the diff highlighting and folds:
+
+`:diffupdate`
+
+- Toggle code fold:
```

Comment:
Perhaps this will read better?
```suggestion
- Toggle code folding:
```

Reply:
This would also be incorrect since it only toggles the highlighted code fold, not all folds. I've updated this line in my new commit to make that more clear however.